src/fragments/lib/logging/common/change_log_levels/change_log_levels.mdx

```diff
@@ -0,0 +1,121 @@
+         <Fragments
+            fragments={{
+                ios: iosDefaultLogLevel,
+                android: androidDefaultLogLevel
+            }}/>
+    </Block>
+</BlockSwitcher>
+
+## Change local storage maximum size
+To change the maximum amount of local logs are stored before sending logs to CloudWatch.
+
+Below is an example of setting the local storage size to 2 MB:
+
+import iosLocalStorageSize from '/src/fragments/lib/logging/ios/change_log_levels/20_local_storage_size.mdx';
+import androidLocalStorageSize from '/src/fragments/lib/logging/android/change_log_levels/20_local_storage_size.mdx';
+import commonLocalStorageSize from '/src/fragments/lib/logging/common/change_log_levels/20_local_storage_size.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonLocalStorageSize}}/>
+    </Block>
+    <Block name="Without Configuration File">
+        <Fragments
+            fragments={{
+                ios: iosLocalStorageSize,
+                android: androidLocalStorageSize
+            }}/>
+    </Block>
+</BlockSwitcher>
+
+## Change automatic log flush interval
+To change the time interval when logs are automatically flushed and sent to CloudWatch.
+
+Below is an example of setting the time interval to 120 seconds:
+
+import iosflushInterval from '/src/fragments/lib/logging/ios/change_log_levels/30_flush_interval.mdx';
+import androidflushInterval from '/src/fragments/lib/logging/android/change_log_levels/30_flush_interval.mdx';
+import commonflushInterval from '/src/fragments/lib/logging/common/change_log_levels/30_flush_interval.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonLocalStorageSize}}/>
+    </Block>
+    <Block name="Without Configuration File">
+        <Fragments
+            fragments={{
+                ios: iosflushInterval,
+                android: androidflushInterval
+            }}/>
+    </Block>
+</BlockSwitcher>
+
+## Configure Log Level by Category
+Each Amplify category can be configure with its own log level.  
+
+Below is an example of setting different log levels for Storage and Auth categories:
+
+import iosCategoryLogLevel from '/src/fragments/lib/logging/ios/change_log_levels/40_category_log_level.mdx';
+import androidCategoryLogLevel from '/src/fragments/lib/logging/android/change_log_levels/40_category_log_level.mdx';
+import commonCategoryLogLevel from '/src/fragments/lib/logging/common/change_log_levels/40_category_log_level.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonCategoryLogLevel}}/>
+    </Block>      
+    <Block name="Without Configuration File">
+        <Fragments
+            fragments={{
+                ios: iosCategoryLogLevel,
+                android: androidCategoryLogLevel
+            }}
+        />  
+    </Block>
+</BlockSwitcher>
+
+## Configure Log Level Log by User
```

Comment:
can we please move this section into a seperate page as well and call it "Configure User Allow List"

Reply:
move to a new page as suggested.